packages/harmony/src/storybook/colors.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { ColorPalette } from './components/ColorPalette/ColorPalette'
```

Comment:
do we not want to try and use mdx for this? also since ColorPalette is not used anywhere except for this story, should we just move it all into here?

Reply:
tsx seems easier to work with to me overall. And I was thinking of splitting up the component in the future to a few other ones that could be used for other foundations pages so a components folder seemed like a good idea

Reply:
yeah agreed tsx feels easier, but it is the case that we have that "color" title, and then the story right, vs mdx it's just the Title yeah? i mean maybe we can just have an mdx file that simply renders the color palette component, would that remove the storybook default title?

Reply:
Update here: trouble with control in storybook docs without a story so leaving mdx for later when we figure out this issue or theming overall